pkgs/by-name/ni/nixos-firewall-tool/package.nix

```diff
@@ -1,15 +1,16 @@
-{ writeShellApplication, iptables, lib }:
+{ writeShellApplication, iptables, lib
+, nftables, backend ? "iptables"
+}:
 
 writeShellApplication {
   name = "nixos-firewall-tool";
+
   text = builtins.readFile ./nixos-firewall-tool.sh;
-  runtimeInputs = [
-    iptables
-  ];
+  runtimeInputs = [ ({ inherit iptables nftables; }.${backend}) ];
```

Comment:
Given that the firewall module have `cfg.package` added to the environment package. Maybe the `runtimeInputs` can be removed completely and rely on `iptables/nft` being in PATH?